Yacht Dice: Textual fixes and changes (Docs / yaml description)

spinerak opened the pull request:
## What is this fixing or adding?
I've changed several textual... texts.
Most importantly, I've changed the setup guide (there was a broken link in there too; ../player-options).
I've made it much simpler, not explaining the entire AP workings.

## How was this tested?
I've tried and tested it via webhost.

## The new setup guide will look like this:
![image](https://github.com/user-attachments/assets/5f2b2c98-453f-4fb6-a2c4-9a3cfae2276d)

spinerak commented:
ah damn I messed up and added other stuff to this PR, hold....

spinerak commented:
I reverted it, ok should be good again......

palex00 commented on worlds/yachtdice/docs/setup_en.md:
```diff
@@ -3,19 +3,13 @@
 Open the Yacht Dice website. There are two options:
-- Download the latest release from [Yacht Dice Release](https://github.com/spinerak/ArchipelagoYachtDice/releases/latest) and unzip the Website.zip. Then open player.html in your browser. 
+- Download the latest release from [Yacht Dice Release](https://github.com/spinerak/ArchipelagoYachtDice/releases/latest) and unzip the Website.zip. Then open index.html in your browser. 
 - Cruise over to the [Yacht Dice website](https://yacht-dice-ap.netlify.app/). This also works on mobile. If the website is not available, use the first option.
```
I would reverse these options so the website (the easier way) is listed first.

Maybe explain why one would prefer the one over the other?

spinerak replied:
Changing the order makes sense!
I'm not sure how to explain why it's easier, I think it follows from the fact it's only one step :)
Thanks for the feedback!

palex00 commented on worlds/yachtdice/docs/setup_en.md:
```diff
@@ -6,10 +6,10 @@
 
 Press Archipelago, and after logging in, you are good to go. The website has a built-in client, where you can chat and send commands.    
-Both options also have an "offline" play option to try out the game without having to generate a game first.
+The website also have an "offline" play option to try out the game without having to generate a game first.
```
Why this change? Is it because the download is "technically" already offline? I'd still say it's fair to say you can do it offline with the downloaded version tbh.


If you want to keep this change though you need to change have to has.

spinerak replied:
Fair! I've changed it to:
Both options also have a "Solo play" mode
Which I think makes sense because that's what the mode is called on the website.

palex00 reviewed (approved):
All concerns were addressed. Looks good. Fixes erronous information in the Options.py, too.

spinerak commented:
I've updated the description of the PR: I've added a screenshot of what the setup page looks to make review easier.
@NewSoupVi would you be able to review this for me please? It's the last open PR for now that would really help give Yacht Dice the best introduction possible :)
